app/src/main/java/protect/card_locker/Utils.java

```diff
@@ -55,6 +55,7 @@
 import java.io.InputStreamReader;
 import java.math.BigDecimal;
 import java.nio.charset.StandardCharsets;
+import java.text.DecimalFormat;
```

Comment:
unused import?

Reply:
Thanks, I forgot to remove it before committing